app/cypress/e2e/0-ndr-core-tests/gp_view_lloyd_george_workflow.cy.js

```diff
@@ -17,77 +16,162 @@ describe('GP View Lloyd George Workflow', () => {
+            assertEmptyLloydGeorgeCard();
+        });
     });
 
-    it('displays an empty Lloyd George card when the backend API call fails', () => {
-        // Act
-        cy.intercept('GET', '/LloydGeorgeStitch*', {
-            statusCode: 500,
+    context('Download Lloyd George document', () => {
+        beforeEach(() => {
+            cy.intercept('GET', '/LloydGeorgeStitch*', {
+                statusCode: 200,
+                body: viewLloydGeorgePayload,
+            }).as('lloydGeorgeStitch');
+            cy.get('#verify-submit').click();
+            cy.wait('@lloydGeorgeStitch');
+        });
+
+        it('allows a GP user to download the Lloyd George document of an active patient', () => {
+            cy.intercept('GET', '/DocumentManifest*', {
+                statusCode: 200,
+                body: 'http://localhost:3000/browserconfig.xml', // uses public served file in place of a ZIP file
+            }).as('documentManifest');
+
+            cy.getByTestId('actions-menu').click();
+            cy.getByTestId('download-all-files-link').click();
+
+            cy.wait('@documentManifest');
+
+            // Assert contents of page when downloading
+            cy.contains('Downloading documents').should('be.visible');
+            cy.contains(
+                `Preparing download for ${viewLloydGeorgePayload.number_of_files} files`,
+            ).should('be.visible');
+            cy.contains('Compressing record into a zip file').should('be.visible');
+            cy.contains('Cancel').should('be.visible');
+
+            // Assert file has been downloaded
+            cy.readFile(
+                `cypress/downloads/lloyd_george-patient-record-${searchPatientPayload.nhsNumber}.xml`,
+            );
+
+            // Assert contents of page after download
+            cy.contains('Download complete').should('be.visible');
+            cy.contains('Documents from the Lloyd George record of:').should('be.visible');
+            cy.contains(
+                `${searchPatientPayload.givenName} ${searchPatientPayload.familyName}`,
+            ).should('be.visible');
+            cy.contains(`(NHS number: ${searchPatientPayload.nhsNumber})`).should('be.visible');
+
+            cy.getByTestId('return-btn').click();
+
+            // Assert return button returns to pdf view
+            cy.getByTestId('pdf-card').should('be.visible');
+        });
+
+        it('displays an error when no Lloyd George record exists for the patient', () => {
+            cy.intercept('GET', '/DocumentManifest*', {
+                statusCode: 204,
+                body: 'No documents found for given NHS number and document type',
+            }).as('documentManifest');
+
+            cy.getByTestId('actions-menu').click();
+            cy.getByTestId('download-all-files-link').click();
+
+            cy.wait('@documentManifest');
+
+            // Assert
+            // TODO - implement error scenario in UI and amend assertion accordingly
```

Comment:
Looks like a required case has not been implemented in dev. From a code perspective this can be merged but do we need to raise a bug to cover this?

Reply:
Sure, I've set these tests to skip for now, and will raise bug tickets for the fix